tests/TestHelpers/WebDavHelper.php

```diff
@@ -898,4 +878,66 @@ public static function getMtimeOfResource(
 		$mtime = new DateTime($xmlPart[0]->__toString());
 		return $mtime->format('U');
 	}
+
+	/**
+	 *
+	 * @param string $property
+	 * @param string $baseUrl
+	 * @param string $user
+	 * @param string $password
+	 * @param string $path
+	 * @param string|null $xRequestId
+	 * @param int|null $davPathVersionToUse
+	 *
+	 * @return string
+	 * @throws GuzzleException
+	 */
+	public static function getFilePropertyForPath(
+		string $property,
+		string $baseUrl,
+		string $user,
+		string $password,
+		string $path,
+		?string $xRequestId = '',
+		?int $davPathVersionToUse = self::DAV_VERSION_NEW
+	): string {
+		$body
+			= "<?xml version='1.0'?>
+				<d:propfind  xmlns:d='DAV:' xmlns:oc='http://owncloud.org/ns'>
+ 					<d:prop>
+ 					    <$property />
+  					</d:prop>
+				</d:propfind>";
+
+		$response = self::makeDavRequest(
+			$baseUrl,
+			$user,
+			$password,
+			"PROPFIND",
+			$path,
+			null,
+			$xRequestId,
+			$body,
+			$davPathVersionToUse
+		);
+
+		$statusCode = $response->getStatusCode();
+		$responseBody = $response->getBody()->getContents();
+
+		if ($statusCode != 207) {
+			throw new Error($responseBody . "\n" . "HTTP status code should be 200 but found $statusCode");
+		}
```

Comment:
```suggestion
                Assert::strictEquals(207, $statusCode, "HTTP status code should be 200 but found $statusCode\nResponse Body:\n$responseBody");
```

Reply:
```suggestion
		if ($statusCode != 207) {
			throw new Error($responseBody . "\n" . "HTTP status code should be 200 but found $statusCode");
		}
```
The status code is compared with `207` but in the message `status code should be 200`. why?